### DISCUSSION TITLE
Tables rowspan / colspan

### DISCUSSION THREAD

**Discussion by oshybrid** [requarks/wiki#7409] [Error / Bug Report]
The various techniques to add rowspans and colspan work when I am editing in the app's Markdown editor.  And they are rendering properly in the preview pane.  But when I save the page the final rendered wiki page does not work.  It does not match the preview rendering.  As in the final rendering does not recognize the various ways to make a row or col span.  Am I missing something?  Again, the preview pane with the Markdown editor is rendering the table as one would expect.

MARKDOWN
| One    | Two | Three | Four    | Five  | Six 
| -
| Span <td colspan=3>triple  <td colspan=2>double

HOW IT IS RENDERED ON PAGE
| One | Two | Three | Four | Five | Six
| -
| Span triple double

MARKDOWN
"| Col 1 | Col 2 | Col 3 | Col 4 |
|---|---|---|---|
| info | info | info | info |
| Colspan ||| info |
| Rowspan | info | Col and rowspan ||
| ^^ | info | ^^ |"

HOW IT IS RENDERED ON PAGE
info | info | info | info
Colspan |   |   | info
Rowspan | info | Col and rowspan |  
^^ | info | ^^

**Comment by clonejo** [comment]
https://github.com/requarks/wiki/discussions/2879#discussioncomment-259454 worked for me.  (Administration->Rendering->enable MultiMarkdown support)

**Reply by amateusz** [reply]
for future generations, since this is an issue since 2021:
to be precise:
enabling `Administration → Rendering → MultiMarkdown Table` enables rendering of `colspan`
enabling `Administration → Rendering → MultiMarkdown Table → Rowspan` enables `rowspan`

**and** you need to visit given md page and save it to trigger a re-rendering

and yes, confusingly this is enabled in the markdown editor preview, even if disabled in the rendering options
